bounties_api/std_bounties/seo_client.py

```diff
@@ -0,0 +1,56 @@
+from bounties import settings
+from bounties.sns_client import sns_publish
+from bounties.utils import base_url_for, bounty_url_for, profile_url_for
+from std_bounties.models import Bounty
+from user.models import User
+from uuid import uuid4
+
+
+class SEOClient:
+
+    def __init__(self):
+        pass
+
+    def publish_new_sitemap(self, platform):
+        if (platform == 'bounties-network' or platform == 'gitcoin') and settings.ENVIRONMENT == 'rinkeby':
```

Comment:
These `if`s are a little hard to follow for me. Wish we didn't have to make such specific checks for gitcoin here.